File: ClientApp/app/model/LiveRequest.ts
export class LiveRequest {
    public conv_id: string;
    public user: string;
    public action: string;
    public date: string;
    constructor(conv_id: string, user: string, action: string) {
        this.conv_id = conv_id;
        this.user = user;
        this.action = action;
        this.date = new Date().toISOString();
    }
}

File: Models/LiveRequest.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace chatbot_portal.Models
{
    public class LiveRequest
    {
        [Key]
        public string conv_id { get; set; }
        public string action { get; set; }
        public string date { get; set; }
        public string user { get; set; }
        //public byte[] last_modified { get; set; }

        /*
        public override string ToString()
        {
            //return "{ conv_id: " + conv_id + ", "
        }
        */
    }
}
